diff --git a/dist/index-3c244051.js b/dist/index-3c244051.js
--- a/dist/index-3c244051.js
+++ b/dist/index-3c244051.js
@@ -26641,7 +26641,7 @@ function markdocNodeToProseMirrorNode$1(node, parentType) {
           location: node.location
         });
       }
-      if (componentConfig.kind === 'inline' && !(parentType !== null && parentType !== void 0 && parentType.isTextblock)) {
+      if (componentConfig.kind === 'inline' && !(parentType !== null && parentType !== void 0 && (parentType.isTextblock || parentType === schema.nodes.table_cell))) {
         return schema.nodes.paragraph.createAndFill({}, pmNode);
       }
       return pmNode;
@@ -27139,7 +27139,7 @@ function markdocNodeToProseMirrorNode(node, parentType) {
       if (!pmNode) {
         error(`${node.type} has unexpected children`);
       }
-      if (componentConfig.kind === 'inline' && !(parentType !== null && parentType !== void 0 && parentType.isTextblock)) {
+      if (componentConfig.kind === 'inline' && !(parentType !== null && parentType !== void 0 && (parentType.isTextblock || parentType === schema.nodes.table_cell))) {
         return schema.nodes.paragraph.createAndFill({}, pmNode);
       }
       return pmNode;
