diff --git a/dist/solana-lib.cjs.development.js b/dist/solana-lib.cjs.development.js
--- a/dist/solana-lib.cjs.development.js
+++ b/dist/solana-lib.cjs.development.js
@@ -1325,7 +1325,7 @@ const HtmlParser = ({
     className: "tw-scroll-smooth"
   }, /*#__PURE__*/React__default.createElement("div", {
     className: `tw-html_parser ${additionalClases}`
-  }, parsedNodes));
+  }, ...parsedNodes));
 };
diff --git a/dist/solana-lib.esm.js b/dist/solana-lib.esm.js
--- a/dist/solana-lib.esm.js
+++ b/dist/solana-lib.esm.js
@@ -1299,7 +1299,7 @@ const HtmlParser = ({
     className: "tw-scroll-smooth"
   }, /*#__PURE__*/React__default.createElement("div", {
     className: `tw-html_parser ${additionalClases}`
-  }, parsedNodes));
+  }, ...parsedNodes));
 };
